terraform/stacks/unimelb/data_archive/byob_ica_v2.tf

```diff
@@ -616,6 +616,36 @@ data "aws_iam_policy_document" "development_data" {
       "${aws_s3_bucket.development_data.arn}/*",
     ]
   }
+  statement {
+    sid = "nextflow_batch"
+    principals {
+      type        = "AWS"
+      // arn:aws:iam::843407916570:role/nextflow/oncoanalyser-pipeline-batch-instance-role
+      identifiers = [
+        "arn:aws:iam::${local.account_id_dev}:role/nextflow-oncoanalyser-pipeline-batch-instance-role",
+        "arn:aws:iam::${local.account_id_dev}:role/nextflow-oncoanalyser-task-batch-instance-role",
+        "arn:aws:iam::${local.account_id_dev}:role/nextflow-sash-pipeline-batch-instance-role",
+        "arn:aws:iam::${local.account_id_dev}:role/nextflow-sash-task-batch-instance-role",
+      ]
+    }
+    actions = [
+      "s3:GetBucketLocation",
```

Comment:
This is needed?

Reply:
portal_data_access changes?

Reply:
> portal_data_access changes?

Just fixed a typo

Reply:
I meant that the last commit changed the permissions of the portal_data_access where I think it should have updated the permissions of the nextflow_batch statement...

Reply:
Oh right, sorry, let me have a look at that

Reply:
Ah so it did, force-pushing